src/data.js

```diff
@@ -0,0 +1,240 @@
+  },
+  {
+    id: 4,
+    embedId: "yLZazznWoAs",
+    title: "JavaScript in the browser!",
+    views: 18120,
+    age: "1 Years",
+    likes: 750,
+    channel: {
+      id: "2",
+      title: "Tanay Pratap",
+      subscribers: "21600",
+      logo:
+        "https://yt3.ggpht.com/ytc/AAUvwnibnmzbi8nfRHEAzdI-8lpPGsrD1F6Cg3mAImtQ=s176-c-k-c0x00ffffff-no-rj-mo",
+    },
+    thumbnailImgUrl:
+      "https://i.ytimg.com/vi/yLZazznWoAs/hqdefault.jpg?sqp=-oaymwEcCNACELwBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCyni0d8rdyR8ngBZzCJ1_mn3qU2w",
+  },
+  {
+    id: 5,
+    embedId: "u6_a0d94A1Q",
+    title: "Getting started with CSS3",
+    views: 16878,
+    age: "1 Years",
+    likes: 427,
+    channel: {
+      id: "2",
+      title: "Tanay Pratap",
+      subscribers: "21600",
+      logo:
+        "https://yt3.ggpht.com/ytc/AAUvwnibnmzbi8nfRHEAzdI-8lpPGsrD1F6Cg3mAImtQ=s176-c-k-c0x00ffffff-no-rj-mo",
+    },
+    thumbnailImgUrl:
+      "https://i.ytimg.com/vi/u6_a0d94A1Q/hqdefault.jpg?sqp=-oaymwEcCNACELwBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCRYdIQvuU6BDf0koL32Yy7CecTog",
+  },
+  {
+    id: 6,
+    embedId: "No8qdcVYiQw",
+    title: "Getting started with HTML5",
+    views: 15818,
+    age: "1 Years",
+    likes: 582,
+    channel: {
+      id: "2",
+      title: "Tanay Pratap",
+      subscribers: "21600",
+      logo:
+        "https://yt3.ggpht.com/ytc/AAUvwnibnmzbi8nfRHEAzdI-8lpPGsrD1F6Cg3mAImtQ=s176-c-k-c0x00ffffff-no-rj-mo",
+    },
+    thumbnailImgUrl:
+      "https://i.ytimg.com/vi/No8qdcVYiQw/hqdefault.jpg?sqp=-oaymwEcCNACELwBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLClk-TuHV313-I3NP6onb3Yzr_nKw",
+  },
+  {
+    id: 7,
+    embedId: "iiADhChRriM",
+    title: "Learn JSON in 10 Minutes",
+    views: 1090128,
+    age: "1 Years",
+    likes: 35087,
+    channel: {
+      id: "3",
+      title: "Web Dev Simplified",
+      subscribers: "498990",
+      logo:
+        "https://yt3.ggpht.com/ytc/AAUvwnhIz_0Su6HhW6Ym50QCroJCAnF10X9xnnMDboN2=s68-c-k-c0x00ffffff-no-rj",
+    },
+    thumbnailImgUrl:
+      "https://i.ytimg.com/vi/iiADhChRriM/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLClZQR43eBXXKvr_odFEvXPc8ZeYg",
+  },
+  {
+    id: 8,
+    embedId: "jx5jmI0UlXU",
+    title: "Learn CSS Position In 9 Minutes",
+    views: 707315,
+    age: "2 Years",
+    likes: 32786,
+    channel: {
+      id: "3",
+      title: "Web Dev Simplified",
+      subscribers: "498990",
+      logo:
+        "https://yt3.ggpht.com/ytc/AAUvwnhIz_0Su6HhW6Ym50QCroJCAnF10X9xnnMDboN2=s68-c-k-c0x00ffffff-no-rj",
+    },
+    thumbnailImgUrl:
+      "https://i.ytimg.com/vi/jx5jmI0UlXU/hqdefault.jpg?sqp=-oaymwEbCKgBEF5IVfKriqkDDggBFQAAiEIYAXABwAEG&rs=AOn4CLBn_nysLus7y_HpRR0bjx1dxmm9TQ",
+  },
+  {
+    id: 9,
+    embedId: "btj35dh3_U8",
+    title: "Callback Functions in JS ft. Event Listeners 🔥",
+    views: 41088,
+    age: "1 Years",
```

Comment:
Hey @hntejas, I am not sure but this property name "age" is not fitting with context i think. I would prefer to have at is "postedAt" or "publishedAt"? It's my opinion and not required to be correct but please think around it as i have implemented it as this.

Reply:
postedAt or publishedAt will usually refer to a date of video being posted. But with my current data setup I donot have a date, hence the term "age". Once the backend is setup, I'll be adding the postedAt and will be deriving the age based on that date.

Reply:
Great idea!